pymc/distributions/continuous.py

```diff
@@ -1347,13 +1347,22 @@ class Exponential(PositiveContinuous):
     ----------
     lam : tensor_like of float
         Rate or inverse scale (``lam`` > 0).
+    scale: tensor_like of float
+        Alternative parameter (scale = 1/lam).
     """
     rv_op = exponential
 
     @classmethod
-    def dist(cls, lam: DIST_PARAMETER_TYPES, *args, **kwargs):
-        lam = pt.as_tensor_variable(floatX(lam))
+    def dist(cls, lam=None, scale=None, *args, **kwargs):
+        if lam is not None and scale is not None:
+            raise ValueError("Incompatible parametrization. Can't specify both lam and scale.")
+        elif lam is None and scale is None:
+            raise ValueError("Incompatible parametrization. Must specify either lam or scale.")
+
+        if scale is not None:
```

Comment:
We were already using scale internally (see super.dist() below) without naming it so. 

So lets flip it: `if scale is None: scale=reciprocal(lam)` instead. Otherwise we are doing two useless `reciprocal` when a user passes scale

Reply:
Yes I noticed that. But we if implement this, then we need to change **return** to: `return super().dist([scale], **kwargs)`

Reply:
Yeah we should do that, not a problem

Reply:
Okay! So now we would be formally using scale and returning it.